api/CcsSso.Shared/Domain/Helpers/UtilityHelper.cs

```diff
@@ -74,12 +74,20 @@ public static bool IsPasswordValidForRequiredCharactors(string password)
 
     public static bool IsUserNameValid(string Name)
     {
+      bool IsRestrictedChar = false;
       try
       {
+        string[] tokens = "+ * % # ! $".Split();
```

Comment:
Can you use normal array with list of item in it string[] cars = {"Volvo", "BMW", "Ford", "Mazda"};